XCUITests/SearchTest.swift

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+private let LabelPrompt = "Turn on search suggestions?"
+private let SuggestedSite = "foobar2000.org"
+
+class SearchTests: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+    
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    private func suggestionsOnOff() {
+        navigator.goto(SearchSettings)
+        app.tables.switches["Show Search Suggestions"].tap()
+    }
+    
+    func testPromptPresence() {
+        navigator.goto(NewTabScreen)
```

Comment:
so you don't need this goto statement if you're using openNewURL() method, because within the method, it already does goto(NewTabscreen).  this would apply to a number of below tests as well.